app/assets/javascripts/battletype/ship.js

```diff
@@ -11,13 +11,35 @@
         }
       },
       velocity: {
-        get: function () { return this._word; },
+        get: function () { return this._velocity; },
         set: function (v) { this._velocity = v; }
       },
       damage: {
         get: function () { return this._damage; },
         set: function (d) { this._damage = d; }
       },
+      height: {
+        get: function () {
+          if (this._height) {
+            return this._height;
+          } else {
+            // FIXME: Find a better way to get this value.
```

Comment:
It won't work. The ship is not yet in the DOM. For now, the way it's done is as follows:

1. We build the ship
2. We set a random position
3. We add the ship to the combat zone.

We don't really need public accessors. Where could I place this? Directly in the `positionY` setter?